views/js/qtiItem/core/Element.js

```diff
@@ -292,10 +290,19 @@ define([
         getRenderer : function(){
             return this.renderer;
         },
-        render : function(){
-
+        /**
+         * Render the element. Arguments are all optional and can be given in any order.
+         * Argument parsing is based on argument type and is done by taoQtiItem/qtiItem/core/helpers/rendererConfig
+         * @param {Renderer} renderer - specify which renderer to use
+         * @param {jQuery} placeholder - DOM element that will be replaced by the rendered element
+         * @param {Object} data - template data for the rendering
+         * @param {String} subclass - ???????????? If somebody knows...
```

Comment:
subclass enables differentiating behaviour of qti standard class used in different context. eg. we could have different rendering for simpleChoice according to where it is being used: simpleChoice.orderInteraction, simpleChoice.choiceInteraction.

Reply:
great, thanks !